File: problems/1302-delete-characters-to-make-fancy-string/delete-characters-to-make-fancy-string.js
/**
 * @param {string} s
 * @return {string}
 */
const makeFancyString = (s) => {
    let res = [...s]
    for (let i = 0; i < res.length; i++) {
        if (res[i] === res[i + 1] && res[i + 1] === res[i + 2]) {
            res[i] = "";
        }
    }
    return res.join("");
};

File: problems/1302-delete-characters-to-make-fancy-string/delete-characters-to-make-fancy-string.py
class Solution:
    def makeFancyString(self, s: str) -> str:
        res = list(s)
        for i in range(len(res) - 2):
            if res[i] == res[i + 1] and res[i + 1] == res[i + 2]:
                res[i] = ""
            
        return "".join(res)